Test/task3.c

```diff
@@ -0,0 +1,106 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+void readAllTheComments(FILE *file, char *data[], int linesRead);
+
+
+int main()
```

Comment:
```suggestion
void readAllTheComments(FILE *file, char *data[], int linesRead);

int main()
```
Две пустые строки нельзя :)